### ISSUE TITLE
Train resnet50 on ImageNet-1k with single-augmentation but only got 62.7%

### ISSUE THREAD

**Issue by Wen1163204547** [facebookresearch/dino#100]
I want to know the performance of DINO with **single augmentation** compared with MoCov2, so I trained DINO with **ResNet50** backbone on ImageNet-1k  for **100 epochs** but only got linear evaluation Top-1 62.7%.

The train script:
python -m torch.distributed.launch --nproc_per_node=8 main_dino.py --arch resnet50 --output_dir outputs/dino.r50.256bs.100e --local_crops_number 0

The eval script:
python -m torch.distributed.launch --nproc_per_node=8 eval_linear.py --arch resnet50 --pretrained_weights outputs/dino.r50.256bs.100e/checkpoint.pth --output_dir outputs/dino.r50.256bs.100e

Test Results:
Test: Total time: 0:04:03 (0.621918 s / it)                                                                                                                                                                 
* Acc@1 62.688 Acc@5 85.454 loss 1.556                                                                                                                                                                      
Accuracy at epoch 90 of the network on the 50000 test images: 62.7%                                                                                                                                         
Max accuracy so far: 62.69%

I want to know why the linear evaluatation result is much lower than MoCov2 200e? Are the super parameters misaligned? Or multi-augmentation strategy is indispensable for DINO？

**Comment by mathildecaron31** [comment]
Hello,

When training with resnet-50 you need to adapt some hyperparameters as explained here: https://github.com/facebookresearch/dino#resnet-50-and-other-convnets-trainings

I would recommend running something like:
```
python -m torch.distributed.launch --nproc_per_node=8 main_dino.py --arch resnet50 --optimizer sgd --weight_decay 1e-4 --weight_decay_end 1e-4  --local_crops_number 0 --global_crops_scale 0.14 1 --data_path /path/to/imagenet/train --output_dir /path/to/saving_dir
```

**Comment by Wen1163204547** [comment]
> Hello,
> 
> When training with resnet-50 you need to adapt some hyperparameters as explained here: https://github.com/facebookresearch/dino#resnet-50-and-other-convnets-trainings
> 
> I would recommend running something like:
> 
> ```
> python -m torch.distributed.launch --nproc_per_node=8 main_dino.py --arch resnet50 --optimizer sgd --weight_decay 1e-4 --weight_decay_end 1e-4  --local_crops_number 0 --global_crops_scale 0.14 1 --data_path /path/to/imagenet/train --output_dir /path/to/saving_dir
> ```

I have tried this setting but only got linear evaluation Top-1 3.9% . 
```
python -m torch.distributed.launch --nproc_per_node=8 main_dino.py --arch resnet50 --optimizer sgd --weight_decay 1e-4 --weight_decay_end 1e-4 --global_crops_scale 0.14 1 --local_crops_scale 0.05 0.14 --data_path /datapath --output_dir /outputs --batch_size_per_gpu 32 --use_fp16 False
```
With this setting, the training loss does not converge well during training. At the end of 100 epoch training, the training loss is 7.739569.
I think the default learning rate(5e-4) is too small for R50 + SGD.

Did I set something wrong？

**Comment by Wen1163204547** [comment]
I tried the more practical setting in https://dl.fbaipublicfiles.com/dino/dino_resnet50_pretrain/args.txt, here is my args:
```
arch='resnet50', batch_size_per_gpu=32, clip_grad=0.0, data_path='my_path', dist_backend='nccl', dist_url='dist_url', drop_path_rate=0.1, epochs=100, freeze_last_layer=1, global_crops_scale=[0.14, 1.0], gpu=0, local_crops_number=0, local_crops_scale=[0.05, 0.14], local_rank=0, lr=0.3, min_lr=0.0048, momentum_teacher=0.996, norm_last_layer=True, num_machines=16, num_workers=10, optimizer='lars', out_dim=60000, output_dir='my_path', patch_size=16, rank=0, saveckp_freq=20, seed=0, teacher_temp=0.07, use_bn_in_head=True, use_fp16=False, warmup_epochs=10, warmup_teacher_temp=0.04, warmup_teacher_temp_epochs=50, weight_decay=1e-06, weight_decay_end=1e-06, world_size=128
```
With this setting, I get linear evaluation Top-1 49%. This is still lower than SSL models such as MoCov2 or BYOL.
